index.js

```diff
@@ -1,232 +0,0 @@
-humanSecretNumber = randomInteger;
-
-//Human Guess Game Sequence
-async function humanGuessGame() {
-  
-  //Present Challenge
-  let humanGuess = await ask(
-    "I am thinking of a secret number between 1 and 100. Try to guess it..."
-  );
-
-  //Player Wins
-  if (humanGuess == humanSecretNumber) {
-    console.log("You win!");
-    process.exit();
-  }
-  console.log(humanGuess);
-
-  //Main Game Loop
-  while (humanGuess !== humanSecretNumber || humanGuess === humanSecretNumber) {
-    
-    //Player Guesses High
-    if (humanGuess < humanSecretNumber) {
-      humanGuess = await ask("Guess higher...");
-    }
-    
-    //Player Guesses Low
-    if (humanGuess > humanSecretNumber) {
-      humanGuess = await ask("Guess lower...");
-    }
-    
-    //Player Wins
-    if (humanGuess == humanSecretNumber) {
-      console.log("You win!");
-      process.exit();
-    }
-  }
-}
-
-//Computer Guess Game Sequence
-async function computerGuessGame() {
-  
-  //Set the Guess Range
-  setRangeMaximum = await ask(
-    "Let's play a game where you pick a secret number and I guess it.\nThe minimum will be 1 but\nyou're going to pick the highest number of the guessing range.\n"
-  );
-
-  //Translates Maximum Range Data Type
-  let rangeMaximum = parseInt(setRangeMaximum);
-  previousGuess = middleValue(rangeMinimum, rangeMaximum);
-
-  //Smart Smaller Guess
-  if (rangeMaximum < 4) {
-    previousGuess = 1;
-  }
-
-  //Maximum Range Log
-  console.log(`The highest possible number is ${rangeMaximum}.\n`);
-
-  //Pick A Number, Any Number
-  let secretNumber = await ask(
-    "Tell my friend your secret number.\nHe's gonna make sure you don't cheat...\n"
-  );
-
-  //Secret Number Log
-  console.log(`You tell them the secret number is ${secretNumber}.`);
-
-  //First Confirmation
-  let confirmation = await ask(
-    `Is your number ${previousGuess} ?, if it is press (y), if not press (n).\n`
-  );
-
-  //Main Game Loop
-  while (
-    confirmation === "h" ||
-    confirmation === "l" ||
-    confirmation === "y" ||
-    confirmation === "n"
-  ) {
-    //Answer Lie Cheat Detector
-    if (previousGuess == secretNumber && confirmation !== "y") {
-      console.log(
-        "My friend told me you cheated, I don't want to play anymore!"
-      );
-      process.exit();
-    }
-
-    //Computer Victory
-    if (confirmation === "y") {
-      console.log("I Win!");
-      process.exit();
-    }
-
-    //The Computer Guessed Wrong
-    else if (confirmation === "n") {
-      confirmation = await ask(
-        "If your number is higher, press (h), if it is lower, press (l)"
-      );
-    }
-
-    //Modify the Low End
-    if (confirmation === "h") {
-      rangeMinimum = previousGuess;
-    }
-
-    //Range Bust Cheat Detector
-    if (
-      confirmation === "l" &&
-      previousGuess < secretNumber
-    ) {
-      console.log(
-        "My friend told me you cheated, I don't want to play anymore!"
-      );
-      process.exit();
-    }
```

Comment:
I did encounter one issue when testing out your game. When I chose the #1 as my secret number, the game narrowed the range down to (range: 1-2) but before the computer guessed 1, I was hit with a cheater console log. A relatively minor change in the logic check here could resolve this small bug.